opm/common/OpmLog/LogBackend.hpp

```diff
@@ -75,6 +79,7 @@ namespace Opm
         int64_t m_mask;
         std::shared_ptr<MessageFormatterInterface> m_formatter;
         std::shared_ptr<MessageLimiter> m_limiter;
+        int64_t m_level;
```

Comment:
Ok, PR is updated according to  the discussion. 

``@jokva: please use an unsigned type. uint64_t's likely to be the one``

Probably this could be done in a separate PR later as @atgeirr suggested?